src/resource/index.ts

```diff
@@ -5,3 +5,6 @@ export { AuthModule } from './auth/auth.module';
 export { UserModule } from './user/user.module';
 export { UserService } from './user/user.service';
 export { User } from './user/entities/user.entity';
+
+// Link
```

Comment:
Is this commented code still in use? If not, please just remove it to keep the code base clean.

Reply:
Yup, still in use, I forgot to tag the module it's importing so added it here.